### PR TITLE
Add `segments` and `segmentcolor` to aesthetic attributes

### PR DESCRIPTION
**Pull request by eliascarv**

### CODE REVIEW

File: src/cartesiangrid.jl [hunk 1]
```diff
@@ -161,14 +159,13 @@ function vizgrid!(plot)
   viz!(plot, mesh,
     color = color,
     alpha = alpha,
-    showfacets = false
+    aes = Aes(segments=false)
```
Comment (eliascarv) [inline src/cartesiangrid.jl:162]: We need forward other `Aes` attributes?

File: src/collection.jl [hunk 2]
```diff
@@ -50,22 +48,22 @@ function Makie.plot!(plot::Viz{<:Tuple{Collection}})
...
     if isempty(bounds[])
       # nothing to be done
     elseif all(ranks[] .== 1)
       # all boundaries are point sets
       points = Makie.@lift mapreduce(collect, vcat, $bounds)
       viz!(plot, (Makie.@lift Collection($points)),
-        color = facetcolor,
-        showfacets = false,
+        color = aes.segmentcolor,
+        aes = Aes(segments=false)
```
Comment (eliascarv) [inline src/collection.jl:60]: We need forward other `Aes` attributes?

File: src/collection.jl [hunk 3]
```diff
@@ -50,22 +48,22 @@ function Makie.plot!(plot::Viz{<:Tuple{Collection}})
...
       )
     elseif all(ranks[] .== 2)
       # all boundaries are geometries
       viz!(plot, (Makie.@lift Collection($bounds)),
-        color = facetcolor,
-        showfacets = false,
+        color = aes.segmentcolor,
+        aes = Aes(segments=false)
```
Comment (eliascarv) [inline src/collection.jl:66]: We need forward other `Aes` attributes?

File: src/utils.jl [hunk 4]
```diff
@@ -20,6 +20,6 @@ function vizmany!(plot, meshes)
     color = colors,
     alpha = alphas,
     colorscheme = colorscheme,
-    showfacets = false,
+    aes = Aes(segments=false)
```
Comment (eliascarv) [inline src/utils.jl:23]: We need forward other `Aes` attributes?

File: src/cartesiangrid.jl (hunk 1 above)
Comment (juliohm) [reply src/cartesiangrid.jl:162]: For now let's just make the minimum changes necessary. We can think about it later.

**Review by juliohm — APPROVED** [review]

**Pull request merged**